Update spanish books

miguelpalazzo opened the pull request:
## What does this PR do?
Add Resource(s)

## For resources
### Description 
Introduce three (3) unexistent programming books to the Spanish list, each covering different topics of interest:
- NoSQL / Redis
- React
- Go

They're available as HTML (online version), and at least a PDF / EPUB version for downloading purposes. They were originally released in English, but made available as translated resources into Spanish, by Raúl Expósito and others.

### Why is this valuable (or not)?
Redis, React and Go are interesting topics to follow by any developer, and very active and on-demand since a few years ago in the community. However, not so many books get translated or even made available in Spanish language, making such knowledge hard to get for many on Spanish-speaking communities.

### How do we know it's really free?
They're released under Attribution-NonCommercial 3.0 Unported license.

### For book lists, is it a book? For course lists, is it a course? etc.
Books.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

eshellman commented on free-programming-books-es.md:
```diff
@@ -174,6 +176,11 @@
 
+### Go
+
+* [El pequeño libro Go](https://raulexposito.com/the-little-go-book-en-castellano.html) (HTML, PDF, EPUB)
```
Please add author names where appropriate.

eshellman commented on free-programming-books-es.md:
```diff
@@ -241,6 +248,7 @@
 #### React
 
 * [Desarrollo de Aplicaciones Web con React.js y Redux.js](https://leanpub.com/react-redux) (PDF, EPUB, MOBI)
+* [SurviveJS - React de aprendiz a maestro](https://es.survivejs.com) (HTML, PDF)
```
Please add author names where appropriate.

eshellman commented on free-programming-books-es.md:
```diff
@@ -270,6 +278,11 @@
 
+#### Redis
+
+* [El pequeño libro Redis](https://raulexposito.com/the-little-redis-book-en-castellano.html) (HTML, PDF, EPUB)
```
Please add author names where appropriate.

miguelpalazzo commented:
I have added the author names, while also updated from the upstream.

Please, let me know if there is anything else to be done! Thanks in advance @eshellman

eshellman commented:
Beautiful!
Thanks, and welcome to free-programming-books!
